.github/workflows/test.yml

```diff
@@ -3,6 +3,4 @@ name: Continuous integration
 jobs:
   test:
     name: run tests and linters
-    uses: viccuad/github-actions/.github/workflows/reusable-test-policy-rust.yml@v1
-    with:
-      artifacthub: false # change to true to check artifacthub-pkg.yml
+    uses: viccuad/github-actions/.github/workflows/reusable-test-policy-rust.yml@v2
```

Comment:
```suggestion
    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v2
```

Reply:
woops! fixed in https://github.com/kubewarden/rust-policy-template/pull/30/commits/2b14e8dd64cacb8c0aafaa34f79a1a885bc0bcab. Thanks for the catch!